python/smqtk/utils/image_utils.py

```diff
@@ -0,0 +1,54 @@
+import io
+import PIL.Image
+
+from smqtk.representation.data_element.file_element import DataFileElement
+
+
+def is_loadable_image(dfe):
```

Comment:
This should move away from ``DataFileElement`` input requirement because nothing in the body of the function requires file-specific functionality. Should just take in a generic ``DataElement`` instance.